docs/core/deploying/native-aot/index.md

```diff
@@ -38,7 +38,7 @@ On Linux, install the compiler toolchain and developer packages for libraries th
   sudo apk add clang build-base zlib-dev
   ```
 
-<!--Add info for macOS-->
+On macOS, install latest [Command Line Tools for XCode](https://developer.apple.com/xcode/download/).
```

Comment:
```suggestion
On macOS, install latest [Command Line Tools for XCode](https://developer.apple.com/xcode/download/). Only .NET 8 is supported.
```

Reply:
How about this so that we don't have to think about it for .NET 9? Native AOT was introduced in 7.

```suggestion
On macOS, install the latest [Command Line Tools for XCode](https://developer.apple.com/xcode/download/). Not supported on .NET 7.
```

Reply:
"Supported on .NET 8+." or "Supported on .NET 8 and later versions." would be more in line with the rest of the docs.

Reply:
```suggestion
On macOS, install the latest [Command Line Tools for XCode](https://developer.apple.com/xcode/download/). Supported on .NET 8 and later versions.
```